tests/appsec/test_fingerprinting.py

```diff
@@ -0,0 +1,65 @@
+from utils import features
+from utils import interfaces
+from utils import rfc
+from utils import weblog
+
+ARACHNI_HEADERS = {"User-Agent": "Arachni/v1.5.1"}
+DD_BLOCK_HEADERS = {"User-Agent": "dd-test-scanner-log-block"}
+
+
+def get_span_meta(r):
+    res = [span.get("meta", {}) for _, _, span in interfaces.library.get_spans(request=r)]
+    assert res, f"no spans found in {r}"
+    return res
+
+
+@rfc("https://docs.google.com/document/d/1DivOa9XsCggmZVzMI57vyxH2_EBJ0-qqIkRHm_sEvSs/edit#heading=h.88xvn2cvs9dt")
+@features.fingerprinting
+class Test_Fingerprinting_Headers:
```

Comment:
Nit: this should be called `..._Header_And_Network` since the other one is called `Endpoint`.

Reply:
done.